ministryofjustice/analytics-platform-control-panel
# Consolidate UI/backend work done for tools catalogue and home reset.

## What

This PR implements the changes suggested by MOJ UI/UX design specialists.

It takes the work from two previous PRs:

* #824  - the "tools catalogue" that allows folks to select the version of tools to deploy. Work done by @xoen.
* #823  - the "Ripley button" that allows users to reset their home directory. Work done by @ntoll and @tomskelley-gjs.

(These PRs are obsolete and can be safely deleted once this PR lands in main.)

Since both pieces of work touched the main page used by users and added complexity in terms of UI items and workflow, it was decided that, before deploying to users, these changes needed revision to ensure the tools page was simple, clean and obvious to users. This is what this PR does.

You can see the result in the following GIF screenie:

![tools_ui](https://user-images.githubusercontent.com/37602/89551986-794fdd00-d803-11ea-860c-cf39233cd01b.gif)

## How to review

1. Ensure the backend changes (deploying the expected versions, resetting the user's home directory) still work as expected.
2. Check the changes to the UI look like the [suggested changes](https://docs.google.com/presentation/d/17ki5DLiwQCrZ5W7ljqb-8PFSihxQaBLD7Bf3cPHALwY/edit?pli=1#slide=id.g869afc015a_0_198)
3. Ensure the behaviour of the buttons works as expected.

This PR is related to [this Trello ticket](https://trello.com/c/egENqsu2/720-design-work-for-ripley-button-software-catalogue).
